cidc_api/models.py

```diff
@@ -689,6 +689,11 @@ class DownloadableFiles(CommonColumns):
     clustergrammer = Column(JSONB, nullable=True)
     ihc_combined_plot = Column(JSONB, nullable=True)
 
+    def publish_artifact_upload(self):
+        from gcloud_client import publish_artifact_upload
```

Comment:
Would be cool if you could resolve that within this PR

Reply:
Agreed. (`this` being the underlying circular import issue this skirts around.)